pkg/controllers/tunnel/tunnel_test.go

```diff
@@ -0,0 +1,144 @@
+package tunnel_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	v1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	fakeEngine "github.com/submariner-io/submariner/pkg/cableengine/fake"
+	fakeClientset "github.com/submariner-io/submariner/pkg/client/clientset/versioned/fake"
+	submarinerClientsetv1 "github.com/submariner-io/submariner/pkg/client/clientset/versioned/typed/submariner.io/v1"
+	informers "github.com/submariner-io/submariner/pkg/client/informers/externalversions"
+	"github.com/submariner-io/submariner/pkg/controllers/tunnel"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog"
+)
+
+const (
+	namespace = "submariner"
+)
+
+var _ = Describe("", func() {
+	klog.InitFlags(nil)
```

Comment:
Shouldn’t this be an `init()` function?

Reply:
This has been brought up in other PRs 😄 It fails if inside an `init` function or inside the main Test function. I forget the exact reason why.